### PR TITLE
Feature/7 getting started pub

### PR DESCRIPTION
**Pull request by kevinmpowell**

### CODE REVIEW

File: doc-site/docs/getting-started.njk [hunk 1]
```diff
@@ -2,32 +2,184 @@
...
+    ### Sketch Setup
+    #### Plugins
+    1. [Sketch Palettes](https://github.com/andrewfiorillo/sketch-palettes)
+      * Download the [Spirit Palette](https://drive.google.com/open?id=1mbjSJYXNjLRJWiJXlkiFMxR5Qw4Ba-dj) and then load using the plugin.
+    2. [Paddy](https://github.com/DWilliames/paddy-sketch-plugin) for buttons to resize automatically based on label length.
+    
+    #### Nudging Preferences
+    * Spirit uses an 8px grid. To make your life easier, go to Sketch > Preferences > Canvas > Nudging: Move objects 8px using Shift-Arrow keys.
+
+
```
Comment (patrickmckowen) [inline doc-site/docs/getting-started.njk]: Add new H3 `### Sketch Resources`

File: doc-site/docs/getting-started.njk [hunk 2]
```diff
@@ -2,32 +2,184 @@
...
+    #### Add Spirit UI Sketch Library
```
Comment (patrickmckowen) [inline doc-site/docs/getting-started.njk]: Change to H4

File: doc-site/docs/getting-started.njk [hunk 3]
```diff
@@ -2,32 +2,184 @@
...
+    This Library allows use of Spirit symbols (buttons, cards, icons, etc) and styles (typography, shadows, etc) across your Sketch documents and have them update to always be in sync.
+    1. Go here to preview and install: [https://sketch.cloud/s/MMLdQ](https://sketch.cloud/s/MMLdQ) 
+    2. Click &ldquo;Add Library to Sketch&rdquo; button in upper right corner.
+    3. After confirming, Sketch > Preferences > Libraries will open automatically and Spirit will be added.
+    4. You will now find Spirit as an option in your Symbols, Text Styles, and Layer Styles.  Learn more about using a Sketch Library: [https://sketchapp.com/docs/libraries/](https://sketchapp.com/docs/libraries/) 
+
+    ### Download Spirit Marketing UI Kit
```
Comment (patrickmckowen) [inline doc-site/docs/getting-started.njk]: Change to H4

**Comment by patrickmckowen** [comment]
@kevinmpowell looks good.  Seeing this doc with the local nav, I made a couple requests to change what is an H3 and what is an H4.

**Comment by jamesmelzer** [comment]
I like @patrickmckowen's heading suggestions. I committed the change, so I could review the 'final' structure. Approved.

**Comment by kevinmpowell** [comment]
@jamesmelzer thanks for implementing @patrickmckowen's feedback for me. 
screen shot 2018-09-10 at 1 31 45 pm

@patrickmckowen can you approve if this looks correct?

**Comment by patrickmckowen** [comment]
@kevinmpowell there are two issues i'm seeing with text color and local nav but those can be separate tasks.  Doc is approved.  thanks!

**Pull request merged**